src/main/resources/templates/macros/report/embeddings.vm

```diff
@@ -51,8 +51,11 @@
 #macro(includeExpandableEmbedding, $embedding, $type, $content, $index)
   #set($index = $index + 1)
   <div class="embedding indention">
-    <a onclick="attachment=document.getElementById('embedding_$embedding.hashCode()'); attachment.className = (attachment.className == 'hidden' ? 'visible' : 'hidden'); return false" href="#">
-      Attachment $index ($type)</a>
-    <div id="embedding_$embedding.hashCode()" class="hidden"><span class="embedding-box">$content</span></div>
+    <div data-toggle="collapse" data-target="#embedding_$embedding.hashCode()" class="collapsable-control">
```

Comment:
ok, but by this change there is no clear UX that this div is expanded

previously the label was presented as link so user knows that this is clickable, steps/hooks/elements has sign that indicates that div is already opened or can be opened

now you need to mouse over to see that this element is expandable